### PR TITLE
feat: Add `alarm-clock` icon

### PR DESCRIPTION
**Pull request by lscheibel**
Screenshot:

image

### CODE REVIEW

**Comment by ericfennis** [comment]
Nice work @lscheibel !
How does it look if the feet are 45 degrees on the clock?

**Comment by lscheibel** [comment]
Yeah you're right, the legs are way too straight. Here are some experiments:

100%
image

Closeup
image


For me it's a tie between B and D :sweat_smile:

**Comment by ericfennis** [comment]
I agree. B or D on al large scale.
Personally I like the D variant, I think it fits better if you look to the "100%" view.

**Comment by dgdavid** [comment]
I vote for C and D

**Comment by lscheibel** [comment]
Alright, D it is :smile:

**Comment by lscheibel** [comment]
New Screenshot
image

File: icons/alarm-clock.svg [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="12" cy="13" r="8"/>
+  <path d="M12 9v4l2 2"/>
+  <path d="M5 3L2 6"/>
+  <path d="M22 6l-3-3"/>
+  <path d="M6 19l-2 2"/>
+  <path d="M18 19l2 2"/>
```
Comment (jletey) [inline icons/alarm-clock.svg:17]: Ah yeah I'm being that guy rn 👀 But if you could add some spacing that'd be great 🤣 Then we can merge this 🚀

```suggestion
  <circle cx="12" cy="13" r="8" />
  <path d="M12 9v4l2 2" />
  <path d="M5 3L2 6" />
  <path d="M22 6l-3-3" />
  <path d="M6 19l-2 2" />
  <path d="M18 19l2 2" />
```

**Comment by jletey** [comment]
Ah so I found out that I can make those changes myself 😂 Anyways thanks for the great work @lscheibel 💪

**Comment by lscheibel** [comment]
Haha no you're absolutely right, gonna be more careful in the future. The feather repo actually has a precommit hook for that. Don't know if I'm a fan of that but we could try that 😅

**Comment by jletey** [comment]
Yeah I'm trying to refactor the build scripts which formatted the code - and let me tell you it's going so amazingly well 🙃

**Pull request merged**